redbot/cogs/mod/mutes.py

```diff
@@ -322,6 +330,14 @@ async def unmute_voice(
             await ctx.send(
                 _("Unmuted {user} in channel {channel.name}").format(user=user, channel=channel)
             )
+            try:
+                await user.move_to(channel)
+            except discord.Forbidden:
```

Comment:
This should be in an `if` checking whether the bot has that permission so that we don't make API request if we already know that it's gonna fail.
```suggestion
                if channel.permissions_for(ctx.me).move_members:
                    await user.move_to(channel)
                else:
                    raise RuntimeError
            except (discord.Forbidden, RuntimeError):
```

Reply:
👍 As you mentioned `_voice_perm_check()` isn't especially better here as it send a message in the text channel that could be misleading/otherwise require "unnecessary" perms.
_I couldn't get permissions_for to work when I was doing it..._